src/lib/stitching/vortex/stitching.ts

```diff
@@ -128,14 +150,16 @@ export const vortexStitchingEnvironment = (localSchema: GraphQLSchema) => ({
           }
 
           try {
-            return await info.mergeInfo.delegateToSchema({
+            const vortexContext = await info.mergeInfo.delegateToSchema({
               schema: vortexSchema,
               operation: "query",
               fieldName: "analyticsPricingContext",
               args,
               context,
               info,
             })
+            vortexContext.artistNames = artist_names
```

Comment:
IMO: worth a comment about this (passing it down from Artwork so it can be used in `appliedFilterDisplay` as a way to work around the resolver only having access to the data in `AnalyticsPricingContext` and not `Artwork`.